packages/sites/ortho-site/webapp/wdkCustomization/js/client/records/Sequences.tsx

```diff
@@ -457,6 +472,90 @@ export function RecordTable_Sequences(
       />
     ) : null;
 
+  const resetProteinFilterButton = (
+    <OutlinedButton
+      text="Reset protein filter"
+      onPress={() => {
+        setProteinFilterIds([]);
+      }}
+    />
+  );
+
+  const updateProteinFilterIds = () => {
+    setProteinFilterIds(highlightedNodes);
+    setHighlightedNodes([]);
+  };
+
+  const proteinFilter = (
+    <PopoverButton
+      buttonDisplayContent={`Proteins${
+        volatileProteinFilterIds.length > 0
+          ? ` (${volatileProteinFilterIds.length})`
+          : ''
+      }`}
+      key={volatileProteinFilterIds.join(':')}
```

Comment:
This isn't needed
```suggestion
```

Reply:
I think it is necessary - or if we remove it we need another way to close the popover after an action button is clicked

![image](https://github.com/user-attachments/assets/5b3a1959-e195-43ec-8650-88983f01a8e3)

Reply:
I see. This could use a comment.

I'm not a big fan of using keys like this, but it does solve this problem. 

There are two ways we can avoid using key, here:
1. Have `PopoverButton` take an `open` prop that can be used to control the internal state.
2. Have `PopoverButton` take a ref that exposes a `close()` and an `open()` method.

Both have their pros and cons. The first option means introducing more state for the consumer, and more state management for the `PopoverButton` component, but it uses props, which is more common. The latter doesn't not introduce new state and works better with event handlers, but custom refs are not nearly as common to work with.

Reply:
I didn't expose an `open()` method just yet. We can grow into that!